railties/lib/rails/generators/app_base.rb

```diff
@@ -75,7 +75,7 @@ def self.add_shared_options_for(name)
 
         class_option :skip_asset_pipeline, type: :boolean, aliases: "-A", default: nil
 
-        class_option :asset_pipeline,      type: :string, aliases: "-a", default: "sprockets",
+        class_option :asset_pipeline,      type: :string, aliases: "-a", default: "propshaft",
```

Comment:
Might be done in a follow-up PR, but are there documentation changes that we'd like to do as well?